kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/util/PppAuthSecrets.java

```diff
@@ -97,7 +98,7 @@ public PppAuthSecrets(String secretsFilename) {
                 }
             } else {
                 // Create an empty file
-                s_logger.info("File does not exist - creating " + this.m_secretsFilename);
+                s_logger.info("File does not exist - creating " + this.secretsFilename);
```

Comment:
Please change it as: logger.info("File does not exist - creating {}", this.secretsFilename);